src/dev/storybook/aliases.ts

```diff
@@ -22,4 +22,5 @@ export const storybookAliases = {
   ui_actions_enhanced: 'x-pack/plugins/ui_actions_enhanced/.storybook',
   observability: 'x-pack/plugins/observability/.storybook',
   presentation: 'src/plugins/presentation_util/storybook',
+  lists: 'x-pack/plugins/lists/.storybook',
```

Comment:
👋

Reply:
It's a coming! Figure it'll help with integration.